pkg/minikube/constants/constants.go

```diff
@@ -154,38 +107,17 @@ var OldestKubernetesVersion = "v1.11.10"
 var ConfigFile = localpath.MakeMiniPath("config", "config.json")
 
 const (
-	// KubeletServiceFile is the path to the kubelet systemd service
-	KubeletServiceFile = "/lib/systemd/system/kubelet.service"
-	// KubeletSystemdConfFile is the path to the kubelet systemd configuration
-	KubeletSystemdConfFile = "/etc/systemd/system/kubelet.service.d/10-kubeadm.conf"
-	// DefaultCNIConfigPath is the path to the CNI configuration
-	DefaultCNIConfigPath = "/etc/cni/net.d/k8s.conf"
-
-	// GuestAddonsDir is the default path of the addons configuration
-	GuestAddonsDir = "/etc/kubernetes/addons"
 	// GuestManifestsDir is where the kubelet should look for static Pod manifests
 	GuestManifestsDir = "/etc/kubernetes/manifests"
```

Comment:
If you want an idea for a follow-up PR, we would like to move all guest constants and related path calculation functions to a new package called `pkg/minikube/vmpath`. This could then be `vmpath.ManifestsDir`

Reply:
done